reviewrot/__init__.py

```diff
@@ -12,5 +15,9 @@ def get_git_service(git):
     """
     if git == "github":
         return GithubService()
+    elif git == "gitlab":
```

Comment:
Looks like `'gitlab'` stuff is in this PR but shouldn't be.  :p  Tricky branches!